3-platform/delivery/boot/pom.xml

```diff
@@ -58,4 +64,22 @@
       </plugin>
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>debug</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
+        <spring.profiles.active>debug</spring.profiles.active>
+      </properties>
+    </profile>
+    <profile>
+      <id>cloud</id>
+      <properties>
+        <spring.profiles.active>cloud</spring.profiles.active>
+      </properties>
+    </profile>
+  </profiles>
```

Comment:
use Spring Profiles capabilities only. 
no need to complicate via maven.
https://www.baeldung.com/spring-profiles -> e.g. as described in `4.4`
```suggestion
```